src/python/WMCore/WMRuntime/Timestamps.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+_Timestamps_
+
+"""
+
+import getopt
+import logging
+import pickle
+import sys
+
+
+# script options
+options = {
+        "reportIn=": "",      # FJR input file
+        "reportOut=": "",     # FJR output file
+        "timeStartSec=": "",  # UNIX seconds of the start
+        "timeEndSec=": "",    # UNIX seconds of the end
+        }
+
+
+def addTimestampMetrics(config, timeStartSec, timeEndSec):
+    """
+    Adjust timestamp metrics of provied FJR data.
+
+    :param config: input FJR data
+    :param timeStartSec: start timestamp in UNIX seconds
+    :param timeEndSec: end timestamp in UNIX seconds
+    """
+    timing = config.section_('Timing')
+    timing.TotalWallClockTime = timeEndSec-timeStartSec
+
+
+def main():
+    """
+    Main function of the script performs business logic:
+    - parse input arguments
+    - read provided input FJR
+    - adjust timestamp metrics of FJR data
+    - write out FJR
+    """
+    try:
+        opts, _ = getopt.getopt(sys.argv[1:], "", list(options.keys()))
+    except getopt.GetoptError as ex:
+        msg = "Error processing commandline args:\n"
+        msg += str(ex)
+        logging.error(msg)
+        sys.exit(1)
+
+    for opt, arg in opts:
+        if opt == "--reportIn":
+            reportIn = arg
+        if opt == "--reportOut":
+            reportOut = arg
+        if opt == "--timeStartSec":
+            timeStartSec = arg
+        if opt == "--timeEndSec":
+            timeEndSec = arg
+
+    # read content of given FJR report file
+    data = {}
+    with open(reportIn, 'rb') as istream:
+        data = pickle.load(istream)
+
+    # adjust FJR data with provided metrics
+    addTimestampMetrics(data, timeStartSec, timeEndSec)
+
+    # write content of FJR back to report file
+    with open(reportOut, 'wb') as ostream:
+        pickle.dump(data, ostream)
```

Comment:
I do not know if the logging record will show up in the expected log file (wmagentJob.log), but I would suggest to add a logging.info here saying that it successfully created the timestamp metric in the job report file ABC.